src/org/ods/services/OpenShiftService.groovy

```diff
@@ -296,4 +296,11 @@ class OpenShiftService {
         return images
     }
 
+    String getOriginUrlFromBuildConfig (String project, String bcName) {
```

Comment:
We could still use the instance variable `project`, right? I know that how it is used from the pipeline, here the `project` is e.g. `foo-cd` and for the other methods it is e.g. `foo-dev`, but that shouldn't matter. You anyway instantiate a new `OpenShiftService` just to call this method, so you can pass the project to the constructor there ...

Reply:
Anyway, I can also do this cleanup later on - I need to work on the OpenShiftService this week anyway.

Reply:
@michaelsauter  - I would leave this to you .. to do this in one big shot ..